libmscore/property.cpp

```diff
@@ -169,6 +169,10 @@ static const PropertyData propertyList[] = {
       { P_ID::LASSO_SIZE,          false, 0,               P_TYPE::SIZE_MM  },
 
       { P_ID::TIME_STRETCH,        false, 0,               P_TYPE::REAL     },
+      { P_ID::ORNAMENT_STYLE,      false, "ornamentStyle", P_TYPE::ORNAMENT_STYLE},
```

Comment:
ahhh
that’s what this terse comment means?

//
// always: propertyList[subtype].id == subtype
//
//


> On 11 May 2015, at 17:39, Nicolas Froment <notifications@github.com> wrote:
> 
> In libmscore/property.cpp <https://github.com/musescore/MuseScore/pull/1946#discussion_r30050097>:
> 
> > @@ -169,6 +169,10 @@ static const PropertyData propertyList[] = {
> >        { P_ID::LASSO_SIZE,          false, 0,               P_TYPE::SIZE_MM  },
> >  
> >        { P_ID::TIME_STRETCH,        false, 0,               P_TYPE::REAL     },
> > +      { P_ID::ORNAMENT_STYLE,      false, "ornamentStyle", P_TYPE::ORNAMENT_STYLE},
> As stated in the comment at the start of the array, the first element here needs to be in the exact same order than the enum class P_ID in property.h.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/1946/files#r30050097>.
>